src/lang/en-US.ts

```diff
@@ -91,6 +91,7 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
     'directives.grant.skipped.message': `The access grant link you submitted was not applied as you already have the requested access.`,
 
     // User in filters for tables
+    'filter.time.label': `Show stats for`,
```

Comment:
If this content has not been approved yet, I am curious if a concise, informative header would be a better fit (e.g., `Stats Range`).

Reply:
That sounds good to me.